### PR TITLE
fix a typo error

### PR DESCRIPTION
**Pull request by kevinetc123**
Signed-off-by: kaiwentan <kaiwentan@harmonycloud.cn>

**- What I did**
fix a typo mistake. formater -> formatter
**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: cli/command/formatter/disk_usage.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ const (
 	uniqueSizeHeader  = "UNIQUE SiZE"
 )
 
-// DiskUsageContext contains disk usage specific information required by the formater, encapsulate a Context struct.
+// DiskUsageContext contains disk usage specific information required by the formatter, encapsulate a Context struct.
```
Comment (allencloud) [inline cli/command/formatter/disk_usage.go]: also found in https://github.com/docker/docker/blob/master/cli/command/formatter/image.go#L23,
could you update that as well?

Comment (kevinetc123) [inline cli/command/formatter/disk_usage.go]: Sorry It's my negligence. I will correct all the 'formater'.

Comment (allencloud) [inline cli/command/formatter/disk_usage.go]: Thanks for your contribution. I saw you fixed that place as well. 
Maybe still something for us to do. Could you squash the two commits in this PR into a single one?

**Review by thaJeztah — CHANGES_REQUESTED** [review]
changes LGTM, but can you squash your commits, as @allencloud asked?

**Comment by kevinetc123** [comment]
ok

**Comment by allencloud** [comment]
Oh, that is much better now. Thanks a lot @kevinetc123

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Pull request merged**